west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: antmicro
+      url-base: https://github.com/antmicro
```

Comment:
This obviously cannot go in like this, please refer to: https://docs.zephyrproject.org/latest/guides/modules.html#submitting-a-new-module first.